packages/client/src/comp/Navbar.tsx

```diff
@@ -19,22 +19,18 @@ const fadeIn = keyframes`
 
 const slideUp = keyframes`
   from {
-    opacity: 1;
     transform: translateY(calc(var(--height) * 0));
   }
   to {
-    opacity: 0;
     transform: translateY(calc(var(--height) * -1));
   }
 `
 
 const slideDown = keyframes`
   from {
-    opacity: 0;
     transform: translateY(calc(var(--height) * -1));
   }
   to {
-    opacity: 1;
```

Comment:
These opacity changes hides the navbar from showing when not needed, for example when the page is loading and the user is in the middle of the page (i.e. `href=#team`).